test/test_stateless.py

```diff
@@ -0,0 +1,120 @@
+import unittest
+
+import torch
+
+import torch.nn.utils._stateless as _stateless
+from torch.testing._internal.common_cuda import TEST_MULTIGPU
+from torch.testing._internal.common_utils import run_tests, TestCase
+
+
+class MockModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.l1 = torch.nn.Linear(1, 1)
+        buffer = torch.ones(1)
+        self.register_buffer('buffer', buffer)
+
+    def forward(self, x):
+        return self.l1(x) + self.buffer
+
+
+class TestStatelessFunctionalAPI(TestCase):
+    def _run_call_with_module(self, module, device='cpu', prefix=''):
```

Comment:
Given that this function only works for the MockModule above, I would name it `_run_call_with_mock_module`